.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: generic
-os: osx
 sudo: required
-env:
-  - HOMEBREW_NO_AUTO_UPDATE=1
+before_install:
+  - sudo add-apt-repository -y ppa:fish-shell/release-2
+  - sudo apt-get update
```

Comment:
Can we update just the repository? Searching randomly I reached this line `sudo update-ppa ppa:fish-shell/release-2`, but I haven't tried it.

Reply:
🕵️

Reply:
@lsunsi, I found some ways to do that, but I guess we'll have to read some more about Linux to get the answer. 

This looks like it works: https://askubuntu.com/questions/65245/apt-get-update-only-for-a-specific-repository

Reply:
PS: I tried what you suggested, didn't work :(.

Reply:
Yeah, for it to work we'd need to install it, but guess it what: it comes from a PPA repository. So we would need to update everything anyway first. ;D

We can tackle this later on if needed.